__tests__/app.test.js

```diff
@@ -24,4 +24,25 @@ describe('/api/topics', () => {
          });
    
     });
-});
\ No newline at end of file
+});
+
+describe('/api', () => {
+    test('GET: returns an object which describes all available endpoints on the API', () => {
+        return request(app).get('/api')
+        .expect(200)
+        .then((response) => {
+           const endpoints = response.body;
+            // console.log(endpoints, "<endpoints")
+           expect(endpoints).toEqual(expect.any(Object));
+
+                for (let endpointInstance in endpoints) {
```

Comment:
This test will never fail, so when you write a new endpoint, how could you check this has been updated?